common.js

```diff
@@ -176,5 +176,13 @@ export function setupPath(newPathEntries) {
   } else {
     newPath = newPathEntries
   }
+  console.log(`Entries added to ${envPath} to use selected Ruby:`)
+  for (const entry of newPath) {
+    if (!cleanPath.includes(entry)) {
```

Comment:
I think we don't need this check, right?
Because those entries are prepended to the PATH, even if they were already in it.

Reply:
I'll remove that check, easy enough

Reply:
Sorry, AFK (sleeping), common at 6:40 A on a weekend.

> `if (!cleanPath.includes(entry)) {`

I don't normally just add 'stuff'.  Can't remember why I had the conditional.  As you noticed, it's kind of pointless under expected/normal conditions, and should be removed, as it may hide image changes.  We should see everything added to Path, since the Actions logging doesn't show `addPath` items...  Thanks